src/arm/96boards.c

```diff
@@ -34,23 +34,31 @@
 #include "common.h"
 #include "arm/96boards.h"
 
-#define DT_BASE "/sys/firmware/devicetree/base"
+#define DT_BASE "/proc/device-tree"
```

Comment:
I have not tried these boards, but this path is incorrect itself. As stated by the Linux ABI, (https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-firmware-ofw),
>  Userspace must not use the `/sys/firmware/devicetree/base` path directly, but instead should follow `/proc/device-tree` symlink. It is possible that the absolute path will change in the future, but the symlink is the stable ABI.

I believe using this path is a common misconception in the community.